cfme/test_framework/appliance_log_collector.py

```diff
@@ -54,6 +54,11 @@ def pytest_unconfigure(config):
         local_dir.ensure(dir=True)
 
         holder = config.pluginmanager.get_plugin('appliance-holder')
+        if holder is None:
```

Comment:
**QUESTION** when exactly is this triggered, i was under the impression the holder should exist at that point in time

Reply:
Agreed, can we get an insight as to why this is happening please?